xnat17/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM tomcat:7-jre7-alpine
+MAINTAINER Thomas Phil <thomas@tphil.nl>
+
+RUN apk add --no-cache postgresql-client curl
+
+# Download XNAT
+ENV XNAT_VER 1.7.3.1
+ENV XNAT_HOME /data/xnat/home
+RUN curl -k -L -o xnat-web-${XNAT_VER}.war https://bintray.com/nrgxnat/applications/download_file?file_path=xnat-web-${XNAT_VER}.war
```

Comment:
Can you use the [Dockerfile `ADD` instruction](https://docs.docker.com/engine/reference/builder/#add) for this download? That would mean you don't have to install `curl`.

Reply:
We can use ADD, but then the user of this image needs a way to get the correct .war in the right location. Putting the war (~180MB) inside this repo sounds like a bad idea to me.

Another solution could be to remove curl near the end of the Dockerfile.

What are your thoughts?